sql.md

```diff
@@ -0,0 +1,100 @@
+#ORDER:
+
+#Find all subjects sorted by subject
+*SELECT subject FROM subjects
+*ORDER BY subjects ASC;
+
+#Find all subjects sorted by location
+*SELECT subject FROM subjects
+*ORDER BY location ASC;
+
+#WHERE
+
+#Find the book Little Women
+*SELECT * FROM books
+*WHERE title = 'Little Women';
+
+#Find all books containing the word Python
+*SELECT * FROM books
+*WHERE title ILIKE '%python%';
+
+#Find all subjects with the location "Main St" sort them by subject
+*SELECT * FROM subjects
+*WHERE location = 'Main St'
+*ORDER BY subject ASC;
+
+#JOINS
+
+#Find all books about Computers list ONLY book title
+*SELECT title FROM books
+*INNER JOIN subjects
+*ON books.subject_id = subjects.id
+*WHERE subject = 'Computers';
+
+#Find all books and display ONLY
+##Book title
+##Author's first name
+##Author's last name
+##Book subject
+*SELECT books.title, authors.first_name, authors.last_name, subjects.subject  
+*FROM books
+*LEFT JOIN authors
+*ON books.author_id = authors.id
+*INNER JOIN subjects
+*ON books.subject_id = subjects.id;
+
+#Find all books that are listed in the stock table
+##Sort them by retail price (most expensive first)
+##Display ONLY: title and price
+*SELECT books.title, stock.retail FROM editions
+*INNER JOIN stock
+*ON editions.isbn = stock.isbn
+*INNER JOIN books
+*On editions.book_id = books.id
+*ORDER BY stock.retail DESC;
```

Comment:
good work, here is a slight alternative for you - select books.title, stock.retail from stock join editions on stock.isbn = editions.isbn join books on editions.book_id = books.id order by stock.retail desc;